guest/klog/api/k8s_references.go

```diff
@@ -0,0 +1,35 @@
+package api
```

Comment:
I will add a header and also note that this is the logic from real klog except dodging some reflection that crashes the tinygo compiler.